### PR TITLE
chore: rename vc export package

### PR DESCRIPTION
**Pull request by rflechtner**
## fixes KILTProtocol/ticket#3060

Renames the vc-export package, as the `legacy-credentials` package now takes care of this; instead, the package just supplies tools to integrate with `@digitalbazaar/vc`.

Also updates the README and applies a rename and slight logic change to the proof purpose to be used with the suite. The rationale here: while other purpose plugins check that the verification method used matches the expected purpose, our proof is not bound to a VM, so we should assert there is no purpose present on the proof. 

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] I have [left the code in a better state](https://deviq.com/principles/boy-scout-rule)
- [x] I have documented the changes (where applicable)
    * Either PR or Ticket to update [the Docs](https://github.com/KILTprotocol/docs)
    * Link the PR/Ticket here

### CODE REVIEW

File: packages/jsonld-suites/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    - issue a `KiltAttestationProofV1` type proof for a `KiltCredentialV1` type VC.
+    - check the revocation status of a `KiltCredentialV1`.
+
+- `NoProofPurpose`
+
+  - `ProofPurpose` class to be used in combination with the attestation proof suite above.
+
+- `Sr25519Signature2020`
+  - A `suite` implementation for creating and verifying sr25519 linked data signatures.
+- `Sr25519VerificationKey2020`
```
Comment (ntn-x2) [inline packages/jsonld-suites/README.md:42]: Empty line?

**Review by ntn-x2 — APPROVED** [review]
LGTM!

**Review by ntn-x2 — APPROVED** [review]
LGTM!

File: packages/jsonld-suites/README.md (hunk 1 above)
Comment (rflechtner) [reply packages/jsonld-suites/README.md:42]: the auto-formatter set it up that way, not sure why

**Pull request merged**